cli/ops/cephadm.py

```diff
@@ -48,6 +50,19 @@ def generate_bootstrap_config(node, config):
         elif file_type == "json":
             json.dump(config, _f)
 
+        elif file_type == "yaml":
+            for spec in config["spec"]:
```

Comment:
Create separate method for this one and move call it inside bootstrap config parsing.

Reply:
Updated